go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -609,3 +459,13 @@ func serviceControllerKey(endpointSlice *discovery.EndpointSlice) (string, error
 	}
 	return fmt.Sprintf("%s/%s", endpointSlice.Namespace, serviceName), nil
 }
+
+// newRateLimiter makes a queue rate limiter. This limits
+// re-queues somewhat more significantly than base qps.
+// by default the base qps is 10, but this is low for our level of scale.
```

Comment:
how did you come to that conclusion? looks like you are setting to 100

Reply:
From running the scale tests; they expect us to be able to create 100's of services a minute. Remember that each service creation is at least 2, often 3 updates: once to create the empty service, then another as the endpoint slice controller creates the slices.

Reply:
OK, so I don't think you should allow the tokens/sec to be configurable, but the bucket size is hardcoded to 500. Either make the bucket configurable arg, or make the bucket be 5*the tokens/sec.

Reply:
sure, makes sense (I think it's burst, not buckets, FWIW).